R-package/src/cmake/modules/FindLibR.cmake

```diff
@@ -48,14 +49,16 @@ function(create_rlib_for_msvc)
       \nDo you have Rtools installed with its MinGW's bin/ in PATH?")
   endif()
 
+  set(R_MSVC_CORE_LIBRARY "${CMAKE_CURRENT_BINARY_DIR}/R.lib" CACHE PATH "R.lib file location")
```

Comment:
I'm sorry, I was mistaken in my previous comment with name. It should follow official guide naming style and start with `LIBR` like all other output variables. https://cmake.org/cmake/help/v3.0/manual/cmake-developer.7.html?highlight=find_package_handle_standard_args#find-modules

Also, I find `location` is a little bit misleading in terms it refers to "directory", IMHO.